Sources/Subprocess/Buffer.swift

```diff
@@ -150,12 +104,53 @@ extension AsyncBufferSequence.Buffer: Equatable, Hashable {
 
     public func hash(into hasher: inout Hasher) {
         self.data.withUnsafeBytes { ptr in
-            let bytes = UnsafeRawBufferPointer(
-                start: ptr,
-                count: self.data.count
-            )
-            hasher.combine(bytes: bytes)
+            hasher.combine(bytes: ptr)
         }
     }
     #endif
 }
+
+// MARK: - DispatchData.Block
+#if canImport(Darwin) || canImport(Glibc) || canImport(Android) || canImport(Musl)
+extension DispatchData {
+    /// Unfortunitely `DispatchData.Region` is not available on Linux, hence our own wrapper
```

Comment:
Nit: either fix the typo "Unfortunitely" -> "Unfortunately" or just remove the word

```suggestion
    /// `DispatchData.Region` is not available on Linux, hence our own wrapper
```

BTW this doesn't explain why we don't use `Dispatch.Region` on Darwin, and only use our own types for non-Darwin platforms